app/api.py

```diff
@@ -203,3 +204,20 @@ async def all_exception_handler(request: Request, exc: Exception):
             "message": "server error",
         },
     )
+
+@API.post("/financial_aid/{profile_id}")
+async def financial_aid(profile_id: str):
+    """Returns the the probability that financial aid will be required.
```

Comment:
Since the functionality has not been implemented yet, it would be helpful to denote that in the docstring as when people start using the API locally, they'll know this endpoint isn't functional yet.